cvat/apps/events/handlers.py

```diff
@@ -58,10 +58,7 @@ def organization_id(instance):
         return instance.id
 
     try:
-        oid = getattr(instance, "organization_id", None)
-        if oid is None:
-            return instance.get_organization_id()
-        return oid
+        return getattr(instance, "organization_id", None)
```

Comment:
Should this attr be enforced on the objects somehow? It's easy to miss it and it won't be noticed.

Reply:
Currently objects that have foreign key to Organization has `organization_id` attribute, for those models that don't have explicit relation with Organization has `get_organization_id` attribute. This logic is used in few places of code: events, webhooks, permissions. Currently I don't know how this can be done without defining organization_id in each model